reference/config_files/conan.conf.rst

```diff
@@ -80,10 +98,22 @@ available in your machine.
 Conan recipes can use the cpu_count() tool to build the library using more than one core.
 
 
+Storage
++++++++
+The ``storage.path`` variable define the path where all the packages will be stored (on Windows, it is recomended to assign it to
+some unit, e.g. map it to X: in order to avoid hitting the 260 chars path name length limit).
```

Comment:
Maybe mention here the ``short_paths`` variable, and also de-activating the Windows indexer